classes/Conf/Metadata/class.xoctMetadataConfigGUI.php

```diff
@@ -222,7 +226,7 @@ protected function loadList(): void
     /**
      * Converts or better say digest the loaded list and replaces the translation to be processed by the plugin.
      *
-     * @param array  $raw_list the raw list that comes from the listprovider endpoints
+     * @param array $raw_list  the raw list that comes from the listprovider endpoints
```

Comment:
Would be nice to remove extra space!

```suggestion
     * @param array $raw_list the raw list that comes from the listprovider endpoints
```